code/core/src/manager/components/sidebar/SidebarBottom.tsx

```diff
@@ -97,13 +99,17 @@ export const SidebarBottomBase = ({ api, notifications = [], status = {} }: Side
   const wrapperRef = useRef<HTMLDivElement | null>(null);
   const [warningsActive, setWarningsActive] = useState(false);
   const [errorsActive, setErrorsActive] = useState(false);
-  const [testProviders, setTestProviders] = useState<TestProviders>(() =>
-    Object.fromEntries(
+  const [testProviders, setTestProviders] = useState<TestProviders>(() => {
```

Comment:
We should rather implement a dedicated `useSessionStorage` hook instead. Like this one:
https://github.com/streamich/react-use/blob/master/src/useSessionStorage.ts

Reply:
Discussed this with @kasperpeulen and @ndelangen. We concluded that there needs to be a larger discussion and proposal to improve how Storybook handles state storage and synchronization. For the time being we're going to stick with this bespoke sessionStorage usage. We'll fix this when we've come to a consensus on how to handle this type of state.